### ISSUE TITLE
Missing bower.json in package

### ISSUE THREAD

**Issue by netes** [tim-evans/ember-plupload#49]
Hi,
I found a small issue with ember-plupload package. After installing with `ember install` and doing some stuff I had to clear `bower_components` and `node_modules`. Then I ran `npm install` and `bower install` but after trying to start Ember server I had an error:
```ENOENT: no such file or directory, stat '/Users/.../tmp/concat_with_maps-input_base_path-AmSIHfJE.tmp/0/bower_components/plupload/js/moxie.js'```

To fix this I had to do `bower install --save plupload` manually. Then I've looked into `ember-plupload` directory in `node_modules` and it seems that package is missing `bower.json` which is probably the cause of `plupload` not being automatically installed as a dependency.

P.S. Thanks for great addon!

**Comment by tim-evans** [comment]
Thanks @netes! I'm not sure if this is a problem, since I'm using a blueprint to add the dependencies to the hosting app's `bower.json` (https://github.com/tim-evans/ember-plupload/blob/master/blueprints/ember-plupload/index.js).

If you run `ember g ember-plupload`, does that add `plupload` and `dinosheets` to your `bower.json`?

Cheers!

**Comment by netes** [comment]
Yup, `ember g ember-plupload` adds `plupload` and `dinosheets` to `bower.json` but `ember install` doesn't :)

**Comment by tim-evans** [comment]
Weird. I'll check that out.

**Comment by happydude** [comment]
I ran into this error also, plupload was missing from bower.json.  To find the problem I reverted bower.json and package.json and then ran ember install ember-plupload again.  The second time running ember install plupload was added to bower.json.

I can't explain why it wasn't there the first time, but I reviewed my console history and I'm sure I ran ember install ember-plupload both times

**Comment by tim-evans** [comment]
:/

**Comment by tim-evans** [comment]
It should do the thing

**Comment by msalahz** [comment]
i got the same issue
